docs/learn/avalanche/intro.md

```diff
@@ -3,55 +3,39 @@ slug: /intro
 sidebar_label: 🔺 What is Avalanche?
 ---
 
-# Introduction
+# What Is Avalanche?
 
-Avalanche is an open-source platform for building decentralized applications (Dapps) in one
-interoperable, highly scalable ecosystem. Avalanche is the first ecosystem designed to accommodate
-the scale of global finance, with near-instant transaction finality.
+Avalanche is an open-source platform for building decentralized applications in one
+interoperable, decentralized, and highly scalable ecosystem. Powered by a uniquely powerful consensus
+mechanism, Avalanche is the first ecosystem designed to accommodate the scale of global finance, with
+near-instant transaction finality. 
 
-Avalanche empowers developers to join a rich ecosystem of blockchains known as Subnets. Subnets
-are subsets of the Avalanche Primary Network that validate a set of blockchains. Each Subnet's
-failures and performance are isolated from other Subnets in the ecosystem, so increased usage on a
-single Subnet doesn't affect other Subnets. Subnets can be permissioned or permission-less, enabling
-arbitrary levels of customization for any use-case.
+## Why Choose Avalanche?
 
-Dapps run on Virtual Machines (VMs), which serve as definitions for blockchain applications.
-Solidity developers can build on Avalanche's implementation of the EVM straight out-of-the box, or
-build their own custom Virtual Machine (VM) for advanced use cases that require more flexibility
-or performance. VMs are supported in multiple languages including Go and Rust, giving developers
-unparalleled amounts of flexibility in their implementation.
+### Blazingly Fast
 
-Avalanche employs a novel consensus algorithm which offers strong safety guarantees, quick finality
-times, and high throughput, without compromising decentralization. Avalanche uses proof-of-stake and
-is incredibly energy-efficient and sustainable compared to proof-of-work blockchains. Avalanche can
-run on easily accessible consumer-grade hardware, and consumes the same amount of energy as 46 US
-households. The entire Avalanche network consumes only 0.0005% of the amount of energy consumed by
-Bitcoin.
+- Avalanche employs the fastest consensus mechanism of any layer 1 blockchain. The unique consensus
+mechanism enables quick finality and low latency: in less than 2 seconds, your transaction is
+effectively processed and verified.
 
-## Building on Avalanche
+### Built to Scale
 
-### Virtual Machines
+- Anywhere from 1,000 to 10,000 transactions are confirmed per second, virtually unaffected by network
```

Comment:
> Anywhere from 1,000 to 10,000 transactions are confirmed per second

On [the 5-23-23 Dev Community call, @ 21:29](https://www.youtube.com/live/HD3vdZzraE8?feature=share&t=1289), POG announced 50K TPS

Reply:
Patrick was speaking specifically about an instance where he tested a hyperSDK implementation. I've talked with Stephen and we decided to remove this statement all together.